core/shared/src/test/scala/cats/parse/ParserTest.scala

```diff
@@ -177,10 +177,24 @@ object ParserGen {
     GenT[Parser0, List[g.A]](g.fa.rep0(min = min, max = max))
   }
 
+  // this makes an integer >= min, but with power law bias to smaller values
+  def biasSmall(min: Int): Gen[Int] = {
```

Comment:
this turns out to really help us find some bugs. Previously, using choose was making it so we almost never selected 0 or 1, which are the common cases for users. This has the effect of making strong bias to 0, but still quite a good possibility of a larger value too.

cc @non something like this would be good to have baked into scalacheck.Gen.